lib/codelog/cli.rb

```diff
@@ -19,16 +19,24 @@ def new(name = 'change')
     end
 
     desc 'release [VERSION] <RELEASE_DATE>', 'Generate new release updating changelog'
-    def release(version_number, release_date =
-                Date.today.strftime(Codelog::Config.date_input_format))
-      Codelog::Command::Release.run version_number, release_date
+    method_option :preview, desc: 'Prints the preview of the specified version release',
+                            aliases: ['-p', '--preview'], type: :boolean
+    def release(version_number,
```

Comment:
@emilianoLeite As you said, here is where we actually choose to preview or not the release file